setup.py

```diff
@@ -35,43 +34,17 @@
         ),
     ],
     install_requires=[
-        "click==8.1.3",
-        "h5py",
-        "microdf_python",
-        "pandas",
-        "pathlib",
         "policyengine-core",
-        "pytest",
-        "pytest-dependency",
+        "microdf-python",
         "pyyaml",
```

Comment:
```suggestion
```

Reply:
`pyyaml` is apparently never used by policyengine-us.